metricbeat/module/system/process_summary/process_summary.go

```diff
@@ -109,6 +112,7 @@ func (m *MetricSet) Fetch() (common.MapStr, error) {
 		"stopped":  summary.stopped,
 		"zombie":   summary.zombie,
 		"unknown":  summary.unknown,
+		"dead":     summary.dead,
```

Comment:
Please, add also a changelog entry regarding this new field.

Reply:
Done in e4c9bab8e90c4151813763c544210eb2a08f491e